src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -138,9 +147,22 @@ const RewardsTile = ({ className }: RewardsTileProps) => {
   const dispatch = useDispatch()
   const rewardIds = useRewardIds()
   const userChallengesLoading = useSelector(getUserChallengesLoading)
+  const currentStepCountOverrides = useOptimisticChallengeCompletionStepCounts()
+  const [haveChallengesLoaded, setHaveChallengesLoaded] = useState(false)
```

Comment:
Can `haveChallengesLoaded` be inferred (and maybe even `userChallengesLoading`)  by checking if `getUserChallenges` is truthy?

Reply:
the empty object {} for user challenges could be tricky cuz edge case of no challenges active (all disabled) or sthg... there's probably other things we can do or additional checks if we wanted to move that direction